core/right.service.js

```diff
@@ -0,0 +1,18 @@
+//const Promise = require('promise');
```

Comment:
Rights as module name is not a good name, related to usage should rather be something like roles

Reply:
yes I will rename it